HTTP, Texto.

```diff
@@ -0,0 +1,14 @@
+HTTP: <h1>; <HelloWorld>
+</reqiered, tex,h1>;
+</lgm link>
+</video<h1>
+<h1> primera imagen en html</h1>
+
+
+Imagen formato </b>:
+</b>
+<video Src= “http.//link
+<div>
```

Comment:
HTTP, Texto.an andrew I think that is posted indiscriminately wgj

Reply:
> HTTP, Texto.an andrew I think that is posted indiscriminately wgj

Reply:
> HTTP, Texto.an andrew I think that is posted indiscriminately wgj